public/index.html

```diff
@@ -10,7 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
```

Comment:
is there any way to add the fonts locally for the case of the offline environment? we can look into that later, can you create an user story for that?

Reply:
yeah we need to do that. I was going to look at that later. i'll create a user story. or I think we can just add it to the verify everything works offline user story that already exists